docs/content/querying/sql.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: doc_page
+---
+# SQL Support for Druid
```

Comment:
do we need a separate page for that? Maybe put it as part of the main querying page?